electron-utils/oauth.ts

```diff
@@ -64,7 +64,12 @@ function getAuthorizationCode(parentWindow: BrowserWindow, repoPermissionLevel:
     authWindow.webContents.on('will-navigate', (event, newUrl) => {
       if (newUrl.startsWith(CALLBACK_URL)) {
         onCallback(newUrl);
+      } else if (newUrl.startsWith(`${BASE_URL}/session`) || (newUrl.startsWith(`${BASE_URL}/login`))) {
```

Comment:
It seems that unlike newer versions, older versions of Electron (before v9.3.0) do not emit the `will-navigate` event when the app navigates to this page in the authorisation process:



![image](https://user-images.githubusercontent.com/35621759/122665773-c4849b80-d1db-11eb-8f3b-176ae3f4095f.png)
![image](https://user-images.githubusercontent.com/35621759/122665785-dc5c1f80-d1db-11eb-90b9-c366d866d8fe.png)

I could not identify the cause of this discrepancy in Electron's changelog. 
I've added another `else-if` branch here so that these authorisation-related navigations can proceed within the authorisation window (unlike navigations to external links).